src/components/Missions/Missions.js

```diff
@@ -0,0 +1,72 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import fetchMissions, { joinMission } from '../../redux/missions/missions';
+import styles from './Mission.module.css';
+
+const Missions = () => {
+  const { mission } = useSelector((state) => state.missionReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!mission.length) {
+      dispatch(fetchMissions());
+    }
+  }, []);
+
+  const handleClick = (id) => {
+    dispatch(joinMission(id));
+  };
+
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <th>Mission</th>
+          <th>Description</th>
+          <th>Status</th>
+          <th> </th>
+        </tr>
+        {mission.map(
+          ({
+            mission_name: missionName,
+            mission_id: missionId,
+            description,
+            reserved,
+          }) => (
+            <tr key={missionId}>
+              <td className={styles['mission-name']}>{missionName}</td>
+              <td className={styles['mission-description']}>{description}</td>
+              <td className={styles['mission-status']}>
+                {' '}
+                <p
+                  className={
+                    reserved ? styles.reserved : styles['not-reserved']
+                  }
+                >
+                  {reserved ? 'Active Member' : 'NOT A MEMBER'}
+                </p>
+              </td>
```

Comment:
- [ ] Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
   - You can use the syntax:
   -  `{ reserved && (  // code for reserved missions )}`
   - `{ !reserved && (  // code for not reserved missions )}`

Reply:
sure @thecodechaser 😄, I'll fix it.